docs/installation/direct-installation.md

```diff
@@ -19,42 +19,47 @@ git checkout main
 
 ```
 
-- Install dependencies: command `./scripts/direct-install/install-dependencies.sh`
+- Install dependencies command: `./scripts/direct-install/install-dependencies.sh`
 
-- Compile the project: command `./scripts/compile.sh`
+- Compile the project command: `./scripts/compile.sh`
```

Comment:
I think this step can be removed, as it is included in the next one. Check with devs.

Reply:
I asked David here: https://flarenetworks.slack.com/archives/C02NURDPAQZ/p1684866333055599